packages/edit-site/src/components/header/index.js

```diff
@@ -62,7 +62,7 @@ export default function Header( { openEntitiesSavedStates } ) {
 
 	return (
 		<div className="edit-site-header">
-			<FullscreenModeClose />
+			<MainDashboardButton.Slot />
```

Comment:
see #22027 

We're moving away from regular slots. Would you mind doing a small refactor here (bubblesVirtually slots don't support children render function

Reply:
@youknowriad Actually I'm not sure how to achieve the same effect with this, given that it relies on children render function to handle the default case when no fills are provided?

Reply:
There are examples of the same use-case in the code base. See here https://github.com/WordPress/gutenberg/blob/master/packages/block-editor/src/components/block-inspector/index.js#L89-L90

Reply:
Thanks, refactored this in https://github.com/WordPress/gutenberg/pull/22179.